# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/features.h
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/features.h
--- a/gfx/angle/checkout/src/libANGLE/features.h	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/features.h	Fri Nov 08 17:06:00 2019 +0000
@@ -55,11 +55,6 @@
 // Controls if our threading code uses std::async or falls back to single-threaded operations.
 #if !defined(ANGLE_STD_ASYNC_WORKERS)
 #    define ANGLE_STD_ASYNC_WORKERS ANGLE_ENABLED
-#endif      // !defined(ANGLE_STD_ASYNC_WORKERS)
-
-// Force thread safety in all of ANGLE by locking a global mutex in every ANGLE entry point.
-#if !defined(ANGLE_FORCE_THREAD_SAFETY)
-#    define ANGLE_FORCE_THREAD_SAFETY ANGLE_DISABLED
-#endif  // !defined(ANGLE_FORCE_THREAD_SAFETY)
+#endif  // !defined(ANGLE_STD_ASYNC_WORKERS)
 
 #endif  // LIBANGLE_FEATURES_H_